metricflow-semantics/requirements-files/requirements.txt

```diff
@@ -1,8 +1,7 @@
 # Always support a range of production DSI versions capped at the next breaking version in metricflow-semantics.
 # This allows us to sync new, non-breaking changes to dbt-core without getting a version mismatch in dbt-mantle,
 # which depends on a specific commit of DSI.
-# bumping to 0.9.3 will cause test failures, so pinning to dev2 until fixed
-dbt-semantic-interfaces==0.9.4.dev0
+dbt-semantic-interfaces>=0.10.0, <0.11.0
```

Comment:
I think we may need to start by loosening this range for MF semantics specifically before we change the range for MF overall, and then we do the core & mantle update, and then we come back and do the MF update

Reply:
Actually, does this mean we would need a core release too? Or would we just pin to a branch/commit of core in MF

Reply:
Good point. We will need an official core release in order to remove the dev DSI dependency from the currently released version of MF. Not top top prio but maybe we can get them to backport this version bump so that we get a core version with it soon!

Reply:
sounds good!